webui/react/src/pages/Settings.tsx

```diff
@@ -24,27 +22,24 @@ interface Params {
 const DEFAULT_TAB_KEY = TabType.Account;
 
 const SettingsContent: React.FC = () => {
-  const { auth: { user } } = useStore();
   const { tab } = useParams<Params>();
   const location = useLocation();
   const [ tabKey, setTabKey ] = useState<TabType>(tab || DEFAULT_TAB_KEY);
   const history = useHistory();
 
-  const rbacEnabled = queryString.parse(location.search).rbac !== undefined;
+  const rbacEnabled = location.pathname.search('rbac-enable') > 0;
 
   const showTabs = useMemo(() => {
     // TODO: Enable tabs for admin once user management finishes.
-    // return user?.isAdmin || rbacEnabled;
     return rbacEnabled;
-  }, [ rbacEnabled, user?.isAdmin ]);
+  }, [ rbacEnabled ]);
 
   const handleTabChange = useCallback((key) => {
     setTabKey(key);
 
     const basePath = paths.settings(key);
-    const query = rbacEnabled ? 'rbac' : '';
-    const newPath = key === DEFAULT_TAB_KEY ? basePath : `${basePath}/${key}`;
-    history.replace(`${newPath}?${query}`);
+    const query = rbacEnabled ? 'rbac-enable' : '';
```

Comment:
**nit:**
- `rbac-enabled` with a `d`?
- instead of calling it `query` maybe just one line it? ```history.replace(`${basePath}${rbacEnabled ? '/rbac-enabled' : ''});```

Reply:
fixed